src/sentry/tsdb/redis.py

```diff
@@ -173,3 +174,74 @@ def get_range(self, model, keys, start, end, rollup=None):
         for key, points in results_by_key.iteritems():
             results_by_key[key] = sorted(points.items())
         return dict(results_by_key)
+
+    def record(self, model, key, values, timestamp=None):
+        self.record_multi(((model, key, values),), timestamp)
```

Comment:
The base class does the opposite, this follows the same pattern established [here in `base.py`](https://github.com/getsentry/sentry/blob/2c6cd363bd5f85e685ddc03591529dc1002deb58/src/sentry/tsdb/base.py#L115-L130) and [in `redis.py`](
https://github.com/getsentry/sentry/blob/2c6cd363bd5f85e685ddc03591529dc1002deb58/src/sentry/tsdb/redis.py#L110-L134).